influxdb/src/query/mod.rs

```diff
@@ -87,21 +90,42 @@ pub trait Query {
     fn get_type(&self) -> QueryType;
 }
 
+pub trait InfluxDbWriteable {
+    fn into_query(self, name: String) -> WriteQuery;
+}
+
+impl InfluxDbWriteable for Timestamp {
+    fn into_query(self, name: String) -> WriteQuery {
+        WriteQuery::new(self, name)
+    }
+}
+
 impl dyn Query {
     /// Returns a [`WriteQuery`](crate::WriteQuery) builder.
     ///
+    /// # Deprecated
```

Comment:
Does it make sense to deprecate this vs remove it completely? I plan on moving the crate to 0.1 when this PR is merged and the initial stable version should be as clean as possible?

Reply:
Well, if you want to bump to 0.1 it probably makes sense to remove deprecated items. If you want I can remove it.

Reply:
I think I will do that in the Prepare Release Commit then